### ISSUE TITLE
Why are not npm packages in the package.json dependencies?  

### ISSUE THREAD

**Issue by micheledisalvatore** [andyet/signalmaster#73]
Hi,
I'm not understanding why you put the package dependencies in the readme and not in the package.json as dependencies. In this way the dependencies versions are not specified and it could not work in the right way.

I made a demo with simplewebrtc. It works if I use the public server, if I use this server I have an issue on handshake response (400 Bad Request).

These my dependencies:

```bash
├── async@1.5.2
├── colors@1.1.2
├── getconfig@2.2.0
├── node-uuid@1.4.7
├─┬ precommit-hook@0.3.10
│ └─┬ jshint@2.3.0
│   ├─┬ cli@0.4.5
│   │ └─┬ glob@7.0.0
│   │   ├─┬ inflight@1.0.4
│   │   │ └── wrappy@1.0.1
│   │   ├─┬ minimatch@3.0.0
│   │   │ └─┬ brace-expansion@1.1.3
│   │   │   ├── balanced-match@0.3.0
│   │   │   └── concat-map@0.0.1
│   │   ├── once@1.3.3
│   │   └── path-is-absolute@1.0.0
│   ├── console-browserify@0.1.6
│   ├─┬ minimatch@0.4.0
│   │ ├── lru-cache@2.7.3
│   │ └── sigmund@1.0.1
│   ├── shelljs@0.1.4
│   └── underscore@1.4.4
├─┬ redis@2.4.2
│ ├── double-ended-queue@2.1.0-0
│ └── redis-commands@1.1.0
├─┬ socket.io@1.3.7
│ ├─┬ debug@2.1.0
│ │ └── ms@0.6.2
│ ├─┬ engine.io@1.5.4
│ │ ├── base64id@0.1.0
│ │ ├── debug@1.0.3
│ │ ├─┬ engine.io-parser@1.2.2
│ │ │ ├── after@0.8.1
│ │ │ ├── arraybuffer.slice@0.0.6
│ │ │ ├── base64-arraybuffer@0.1.2
│ │ │ ├── blob@0.0.4
│ │ │ └── utf8@2.1.0
│ │ └─┬ ws@0.8.0
│ │   ├─┬ bufferutil@1.2.1
│ │   │ ├── bindings@1.2.1
│ │   │ └── nan@2.2.0
│ │   ├── ultron@1.0.2
│ │   └─┬ utf-8-validate@1.2.1
│ │     └── nan@2.2.0
│ ├─┬ has-binary-data@0.1.3
│ │ └── isarray@0.0.1
│ ├─┬ socket.io-adapter@0.3.1
│ │ ├── debug@1.0.2
│ │ ├── object-keys@1.0.1
│ │ └─┬ socket.io-parser@2.2.2
│ │   └── debug@0.7.4
│ ├─┬ socket.io-client@1.3.7
│ │ ├── backo2@1.0.2
│ │ ├── component-bind@1.0.0
│ │ ├── debug@0.7.4
│ │ ├─┬ engine.io-client@1.5.4
│ │ │ ├── component-inherit@0.0.3
│ │ │ ├── debug@1.0.4
│ │ │ ├─┬ has-cors@1.0.3
│ │ │ │ └── global@2.0.1
│ │ │ ├── parsejson@0.0.1
│ │ │ ├── parseqs@0.0.2
│ │ │ ├── parseuri@0.0.4
│ │ │ ├── ws@0.8.0
│ │ │ └── xmlhttprequest@1.5.0
│ │ ├── has-binary@0.1.6
│ │ ├── indexof@0.0.1
│ │ ├── object-component@0.0.3
│ │ ├─┬ parseuri@0.0.2
│ │ │ └─┬ better-assert@1.0.2
│ │ │   └── callsite@1.0.0
│ │ └── to-array@0.1.3
│ └─┬ socket.io-parser@2.2.4
│   ├── benchmark@1.0.0
│   ├── component-emitter@1.1.2
│   ├── debug@0.7.4
│   └── json3@3.2.6
├─┬ socket.io-client@0.9.16
│ ├─┬ active-x-obfuscator@0.0.1
│ │ └── zeparser@0.0.5
│ ├── uglify-js@1.2.5
│ ├─┬ ws@0.4.32
│ │ ├── commander@2.1.0
│ │ ├── nan@1.0.0
│ │ ├── options@0.0.6
│ │ └── tinycolor@0.0.1
│ └── xmlhttprequest@1.4.2
├─┬ tape@2.14.0
│ ├── deep-equal@0.2.2
│ ├── defined@0.0.0
│ ├─┬ glob@3.2.11
│ │ └── minimatch@0.3.0
│ ├── has@0.0.1
│ ├── inherits@2.0.1
│ ├── object-inspect@1.1.0
│ ├── resumer@0.0.0
│ └── through@2.3.8
├── underscore@1.8.3
└── yetify@0.0.1
```

**Comment by Wasser-Eis** [comment]
> I'm not understanding why you put the package dependencies in the readme and not in the package.json as dependencies. In this way the dependencies versions are not specified and it could not work in the right way.

I have the same question

**Comment by xdumaine** [comment]
What are you talking about? The package.json has dependencies listed, and the readme does not.

**Comment by Wasser-Eis** [comment]
Out off the readme:
"1) npm install async, node-uuid, redis, underscore, precommit-hook, getconfig, yetify, socket.io"

**Comment by fippo** [comment]
fixed in d115b427ea6a4c5d0857e637b5d1b0212c440015
